coordinator/tests/verify-prescription-signatures.spec.ts

```diff
@@ -25,19 +27,21 @@ test.skip("verify signature for specific prescription", () => {
   expectSignatureIsValid(prescriptionRoot)
 })
 
-const cases = specification.map(examplePrescription => [
-  examplePrescription.description,
-  examplePrescription.hl7V3Message
-])
+const cases = fetcher.convertExamples
+  .filter(e => e.isSuccess)
+  .filter(e => e.description.includes("200-OK send"))
+  .map(convertExample => [
+    convertExample.description,
+    readXml(convertExample.response)
+  ])
```

Comment:
Now we can use shared models code in coordinator and smoke-tests,  we can load all convert examples and check their digests and signatures, I've left these as warnings until we work out how we want to approach signing them locally. Smoke tests regenerate ids, sign them in pipeline and verify signatures for each of them at deploy time, so maybe there's some redundancy here

Reply:
After discussing with Dan, these tests probably still have relevance but until we figure out how we want to sign things locally will leave these as warnings. Can reenable them once we've fleshed out local signing process